doc/sources/installation/installation-rpi.rst

```diff
@@ -125,13 +125,22 @@ _`Raspberry Pi 1-4 installation` on Raspbian Jessie/Stretch/Buster
     # or to recompile all files
     make force
 
-   Or on a Raspberry Pi 4 it is possible to use a precompiled wheel. The precompiled wheel can be downloaded from the latest `release <https://github.com/kivy/kivy/releases>`_. A wheel is also automatically build daily and can be downloaded here: `<https://kivy.org/downloads/ci/raspberrypi/kivy>`_.
+   It is also possible to use a precompiled wheel. The precompiled wheel can be downloaded from the latest `release <https://github.com/kivy/kivy/releases>`_. A wheel is also automatically build daily and can be downloaded here: `<https://kivy.org/downloads/ci/raspberrypi/kivy>`_.
 
-   The wheel can be installed as follows::
+   First install the wheel dependency::
 
     python3 -m pip install --upgrade --user wheel
+
+   Now simply install the wheel::
+
     python3 -m pip install --user *armv7l.whl
 
+   It is also possible to install the latest release like so::
+
+    python3 -m pip install --user --extra-index-url https://kivy.org/downloads/simple kivy[base]
+
+   Note that you can add the ``--pre`` argument to install the latest development version.
```

Comment:
We don't link the recent kivy release on the server pypi server. So using `--pre` is pretty much a requirement, not optional when installing from the server.

Reply:
Okay. I'll fix that.

Reply:
Fixed in https://github.com/kivy/kivy/pull/6662/commits/8f6bbd8eb77c48f0474fdbf4232899303b56332c